ci/pipelines/pipeline.yml

```
@@ -0,0 +1,483 @@
+  tags:
+  - resources
+  type: registry-image
+
+resources:
+- check_every: 1h0m0s
+  icon: android
+  name: android-build-box
+  public: true
+  source:
+    repository: mingc/android-build-box
+    tag: 1.24.0
+  tags:
+  - resources
+  type: registry-image
+
+- icon: github
+  name: apiary-mobile
+  public: true
+  source:
+    branch: main
+    password: (("github.com"/token.token))
+    uri: https://github.com/RoboJackets/apiary-mobile
+    username: x-access-token
+  tags:
+  - resources
+  type: git
+  webhook_token: ((webhook-token))
+
+- icon: source-pull
+  name: pull-request
+  public: true
+  source:
+    access_token: (("github.com"/token.token))
+    base_branch: main
+    repository: RoboJackets/apiary-mobile
+    disable_forks: true
+  tags:
+  - resources
+  type: pull-request
+  webhook_token: ((webhook-token))
+
+- name: tagged-release
+  type: git-tag
+  icon: tag
+  source:
+    uri: https://github.com/RoboJackets/apiary-mobile
+    username: x-access-token
+    password: (("github.com"/token.token))
+    tag_filter: v*
+  tags:
+    - resources
+
+- name: tests-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Apiary Android Tests
+    resource_name: tests-check
+  tags:
+    - resources
+
+- name: pr-build-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Pull Request Build
+    resource_name: pr-build-check
+  tags:
+    - resources
+
+- name: internal-test-upload-check
+  type: github-check
+  icon: check
+  public: true
+  source:
+    check_name: Internal Test Draft Release
+    resource_name: internal-test-upload-check
+  tags:
+    - resources
+
+- icon: webhook
+  name: webhooks
+  public: true
+  source: null
+  tags:
+  - resources
+  type: github-webhook
+
+- icon: slack
+  name: slack
+  public: true
+  source: null
+  tags:
+  - resources
+  type: slack
+  
+
+jobs:
+  - name: build-release
+    on_abort:
+      inputs: []
+      params:
+        alert_type: aborted
+        mode: normal
+      put: slack
+    on_error:
+      inputs: []
+      params:
+        alert_type: failed
+        message: Task error, review logs
+        mode: normal
+      put: slack
+    on_failure:
+      inputs: []
+      params:
+        alert_type: failed
+        mode: normal
+      put: slack
+    on_success:
+      inputs: []
+      params:
+        alert_type: success
+        mode: normal
+      put: slack
+    public: true
+    plan:
+      - in_parallel:
+          steps:
+            - get: tagged-release
+              trigger: true
+            - get: android-build-box
+              tags:
+                - large
+              timeout: 5m
+            - do:
+              - task: get-commit-sha
+                config:
+                  platform: linux
+                  image_resource:
+                    type: registry-image
+                    source:
+                      repository: alpine/git
```

Add `tag` to speed up checks

```suggestion
                      repository: alpine/git
                      tag: latest
```